### PR TITLE
Add type="button" to login button

### PR DESCRIPTION
**Pull request by nhducit**
this will prevent form was auto submited

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-facebook-login",
+  "name": "@nhducit/react-facebook-login",
```
Comment (keppelen) [inline package.json]: Why this?

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/keppelen/react-facebook-login.git"
+    "url": "git@github.com:nhducit/react-facebook-login.git"
```
Comment (keppelen) [inline package.json]: Why this?

File: src/facebook.js [hunk 3]
```diff
@@ -198,6 +198,7 @@ class FacebookLogin extends React.Component {
           className={`${cssClass} ${size}`}
           style={ buttonStyle }
           onClick={this.click}
+          type="button"
```
Comment (keppelen) [inline src/facebook.js]: add this on `props`

**Comment by keppelen** [comment]
#98

File: package.json (hunk 1 above)
Comment (nhducit) [inline package.json]: sorry, I forked your repo to use in my project. Please ignore this line

File: package.json (hunk 2 above)
Comment (nhducit) [inline package.json]: sorry, I forked your repo to use in my project. Please ignore this line

**Pull request closed without merging**